src/schema/rules/instruments.yaml

```diff
@@ -1,17 +1,15 @@
 ---
-# This file defines which data types fall under each modality.
+# This file defines which data types fall under each instrument.
 mri:
   datatypes:
   - anat
   - dwi
   - fmap
   - func
   - perf
-eeg:
+bioamp:
   datatypes:
   - eeg
-ieeg:
-  datatypes:
   - ieeg
```

Comment:
This is a core change- now EEG and iEEG are both considered "biopotential amplification". At some point it would be nice to reorganize the specification to reflect this, but I don't think it's necessary.